.changeset/eighty-turkeys-marry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vue-storefront/sdk": major
```

Comment:
Why major?

Reply:
Changed to patch, I thought it does not matter if we are on the prerelease mode.